resources/js/profile-page/daily-challenge.tsx

```diff
@@ -122,26 +123,30 @@ export default class DailyChallenge extends React.Component<Props> {
       return null;
     }
 
+    const playedToday = this.props.stats.last_update != null && moment.utc(this.props.stats.last_update).isSame(Date.now(), 'day');
+
     return (
       <div
         ref={this.valueRef}
-        className='daily-challenge'
+        className={classWithModifiers('daily-challenge', { 'played-today': playedToday })}
         onMouseOver={this.onMouseOver}
       >
-        <div className='daily-challenge__name'>
-          {trans('users.show.daily_challenge.title').split('\\n').map((line, i) => (
-            <div key={i}>{line}</div>
-          ))}
-        </div>
-        <div className='daily-challenge__value-box'>
-          <div
-            className='daily-challenge__value'
-            style={tierStylePlaycount(this.props.stats.playcount)}
-          >
-            {trans(
-              'users.show.daily_challenge.unit.day',
-              { value: formatNumber(this.props.stats.playcount) },
-            )}
+        <div className='daily-challenge__content'>
```

Comment:
The reason for the existence of this `div` is that I need the `::before` pseudo-element that is the check in the corner to be excluded from the flexbox, it takes space otherwise. If you have better ideas on how to implement this I'm open to changes.

Reply:
absolute-positioned element, which the element is, doesn't take space...?

Reply:
Huh. I guess I must have not tried some particular combination of css incantations because it does look to be not required now??? I love css.